next.config.js

```diff
@@ -43,6 +43,11 @@ module.exports = {
         destination: "/lists/:listId/overview",
         permanent: true,
       },
+      {
```

Comment:
I didn't bother adding this redirect in the middleware as it was a simple redirect.